### DISCUSSION TITLE
CMI requests

### DISCUSSION THREAD

**Discussion by Th3v0x** [Zrips/CMI#7901] [Feature Requests]
The plugin is crazy but I have some recommendations that maybe you could consider.
You could add something like a sort of geoip to locate from where each user connects to the server.
Also would be nice the /lay command that the functionality is for the user to lie down.
Add support with MiniMessage to color the configuration messages for example color:#55FF55gradient:blue:red.
Thanks for reading me and keeping me in mind.

**Comment by CloudeLecaw** [comment]
First of all - its never a good idea to create a combinated Request collection in one discussion.
I suggest make one for each one and give them a more usefull title - "cmi requests" was a bad choice.

----------------------------------------------------------------------

> You could add something like a sort of geoip to locate from where each user connects to the server.

geoip allready exists in cmi You need to enable it in config to work.
![Snag_f736ed](https://github.com/Zrips/CMI/assets/12080017/acf48ff1-e9f0-48fe-8ed9-fc414adf823d)

Files you get on the wiki.
![image](https://github.com/Zrips/CMI/assets/12080017/4f747c3c-ef18-47d4-92f4-f3e9819f4a1f)

to see them use:
Use cmi info comamnd, or cmis placeholders
```
%cmi_user_country% – Users country from geoip feature. Example: Germany
%cmi_user_country_code% – Users country code from geoip feature. Example: UK
%cmi_user_city% – Users city name from geoip feature
```
--------------------------------------------------------------------------------------------

> Also would be nice the /lay command that the functionality is for the user to lie down.

 /lay was allready requested in other request https://github.com/Zrips/CMI/discussions/5375
There are even more then this one where /lay was suggested allready...

--------------------------------------------------------------------------------------------
> Add support with MiniMessage to color the configuration messages for example color:#55FF55gradient:blue:red

what is a MiniMassage?

May it worth to ask the Dev of the Minimessage plugin to add gradiant support from CMI you can see it here how cmi handles hex and gradient.
https://www.zrips.net/cmi/hex-colors/
